### PR TITLE
Added Dom Element Type in Third Party Modules GitHub

### PR DESCRIPTION
**Pull request by mtwzim**

### CODE REVIEW

**Comment by mtwzim** [comment]
Solved conflicts

File: database.json [hunk 1]
```diff
@@ -982,6 +982,12 @@
     "repo": "dodecasaurus",
     "desc": "Dice utility that makes generating random dice values simple and convenient."
   },
+  "domelementtype": {
+    "type": "github",
+    "owner": "DenoBRComunitty",
+    "repo": "domelementtype",
+    "desc": "All the types of dom for Deno"
+  },
```
Comment (alirealasad) [inline database.json:990]: put this after dom_serializer.

Comment (alirealasad) [reply database.json:990]: oh wait you already did

Comment (mtwzim) [reply database.json:990]: resolved, I changed the name of the module I added in https://github.com/denoland/deno_website2/pull/887

**Pull request merged**